recipe/meta.yaml

```diff
@@ -16,17 +16,16 @@ build:
 
 requirements:
   build:
-    - pep517
-    - setuptools_scm
-  host:
+    - hatch
```

Comment:
you got to this before I had time... but fyi @goanpeca, projects using hatch will generally only need `hatchling` as a build requirement.  `hatch` itself is a full cli program aimed at developers (but comes with [all these dependencies](https://github.com/pypa/hatch/blob/master/pyproject.toml#L40-L56)).  `hatchling` is all that's required to build

Reply:
Hi @tlambert03 thanks for the info! Will take into account for future builds :)